baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -50,17 +50,17 @@ Leading and trailing whitespaces (`OWS`) are allowed and are not considered to b
 
 #### value
 
-A value contains a string whose character encoding MUST be UTF-8 [[Encoding]].
-Any characters outside of the `baggage-octet` range of characters MUST be percent-encoded.
-The percent character MUST be percent-encoded.
-Characters which are not required to be percent-encoded MAY be percent-encoded.
+A value contains a UTF-8 encoded string.
+Any code points outside of the `baggage-octet` range MUST be percent-encoded.
```

Comment:
let's add code point to glossary? I will forget what it means in a year

Reply:
Do we need to? It is not a concept we are introducing, and there seems to be public documentation on it.

Reply:
I agree with Kalyana. I believe it's pretty clear from context what it means and it only shows up in a single place. Code point is well defined by the unicode spec. We could link `UTF-8` to that?

Reply:
this way it is clearer for me. Thx